crates/scheduler/src/utils.rs

```diff
@@ -243,15 +244,16 @@ pub fn get_time_from_delta(delta: Option<i32>) -> Option<time::PrimitiveDateTime
     delta.map(|t| common_utils::date_time::now().saturating_add(time::Duration::seconds(t.into())))
 }
 
-pub async fn consumer_operation_handler<E>(
-    state: AppState,
+pub async fn consumer_operation_handler<E, T: Send + Sync + 'static>(
```

Comment:
This should also have `AppState*` as trait bound?

Reply:
Changed it to `SchedulerAppState`